docs/release/updating-to-osg-35.md

```diff
@@ -149,6 +149,76 @@ To update OSG Configure on your HTCondor-CE, perform the following steps:
 1.  OSG Configure will warn about config options that it does not recognize;
     delete these options from the config to get rid of the warnings.
 
+Updating to HTCondor-CE 5
+-------------------------
+
+!!!tip "Where to find HTCondor-CE 5"
+
+    [HTCondor-CE 5](https://htcondor.github.io/htcondor-ce/v5/releases/), a major
+    version upgrade from HTCondor-CE 4,
+    is available through the [OSG upcoming](release_series.md) repository.
+
+To update HTCondor-CE, perform the following steps:
+
+1.  Merge any `*.rpmnew` files in `/etc/condor-ce/`
+
```

Comment:
This is probably the most important step to inform folks about: https://htcondor.github.io/htcondor-ce/v5/releases/#new-condor_mapfile-format-and-locations

Reply:
I like this idea. Incorporated it.